src/Workleap.OpenApi.MSBuild/HttpClientWrapper.cs

```diff
@@ -13,14 +13,14 @@ public async Task DownloadFileToDestinationAsync(string url, string destination,
 
         try
         {
-            await using var responseStream = await this._httpClient.GetStreamAsync(url);
+            using var responseStream = await this._httpClient.GetStreamAsync(url);
```

Comment:
Why did we remove the cancellationToken and shouldn't it be passed here? I'm guessing if the user cancels the build it'll request a cancellation on the token and we'd like to cancel the request

Reply:
.NET 2.0 APIs don't support `CancellationToken` parameter